projects/project3/README.md

```diff
@@ -106,7 +107,7 @@ Besides the questions you're expected to answer, you will also be evaluated acco
 		- 0.3 point : at least 25% correct specifications.
 		- 0 point : less than 25% correct specifications.
 
-Note that your implementation might be tested on other layouts.
+Note that your implementation might be tested on other layouts with moving Pacman.
```

Comment:
again, not sure i understand

Reply:
or rephrase as "Note that your implementation might be tested on other layouts, with Pacman moving abitrarily.